bigtable/tests/unit/test_row.py

```diff
@@ -190,31 +180,23 @@ def __init__(self, *args, **kwargs):
                 self._args = []
                 self._kwargs = []
 
-            # Replace the called method with one that logs arguments.
-            def _delete_cells(self, *args, **kwargs):
-                self._args.append(args)
-                self._kwargs.append(kwargs)
-
         row_key = b'row_key'
-        column = b'column'
-        column_family_id = u'column_family_id'
+        column = 'column'
+        column_family_id = 'column_family_id'
```

Comment:
The previous diff (with `b''` and `u''`) is correct.

Reply:
@aneepct: ping.